virttest/utils_sys.py

```diff
@@ -74,6 +75,32 @@ def get_host_bridge_id(session=None):
     return host_bridges if host_bridges else []
 
 
+def is_image_mode(check_command="rpm-ostree", session=None):
+    """
+    Check if current OS is in image mode or package mode
+
+    :param check_command: str, the command to check the mode
+    :param session: aexpect session
+    :return: boolean, True for image mode, otherwise False
+    """
+    if session:
+        status = session.cmd_status(check_command)
+        ret = status == 0
```

Comment:
here only execute cmd rpm-ostree, the return code would be 1, the cmd itself could not run only itself. I suggest using "which rpm-ostree". ("rpm-ostree -v" will also report error)

Reply:
thanks for finding the mistake. Updated. @liang-cong-red-hat